homeassistant/components/zha/__init__.py

```diff
@@ -26,7 +26,7 @@
     DEFAULT_BAUDRATE,
     DEFAULT_RADIO_TYPE,
     DOMAIN,
-    ENABLE_QUIRKS,
+    CONF_ENABLE_QUIRKS,
```

Comment:
`isort` should have been run last, after the other changes.

Reply:
Doh, Good point.

Reply:
I'll fix it in subsequent PR